v2/bigtable-cdc-to-hbase/src/test/java/com/google/cloud/bigtable/data/v2/models/ChangeStreamMutation.java

```diff
@@ -0,0 +1,342 @@
+/*
```

Comment:
You don't have to copy the whole class to test will. You can do something like this: https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/src/test/java/com/google/cloud/spanner/FakePartitionFactory.java#L38

Reply:
We can't create a fake object because Beam transform expects the package name to match 
 package com.google.cloud.bigtable.data.v2.models. 
 Should I strip down the class and remove all non-used functions?

Reply:
Done